text/0131-pass-native-sdk-spans-to-hybrid.md

```diff
@@ -0,0 +1,174 @@
+- Start Date: 2024-02-08
+- RFC Type: feature
+- RFC PR: https://github.com/getsentry/rfcs/pull/131
+- RFC Driver: [Philipp Hofmann](https://github.com/philipphofmann)
+- RFC Status: draft
+
+# Summary
+
+This RFC aims to find a solution for passing auto-instrumented native SDK spans up to the hybrid SDKs.
+
+# Motivation
+
+The native SDKs instrument file IO operations, DB queries, HTTP requests, and more, but this
+information doesn't end up in the hybrid transactions or anywhere else. Our hybrid SDKs instrument
+most of the mentioned operations on the hybrid layer, but they lack information on how long the
+actual native operations take. It could happen that one DB query from the native layer gets
+converted into multiple native DB queries. Such information would be insightful for our users.
```

Comment:
Is this correct?
```suggestion
actual native operations take. It could happen that one DB query from the hybrid layer gets
converted into multiple native DB queries. Such information would be insightful for our users.
```

Reply:
Good catch thanks